modules/plugins/notes/neorg/neorg.nix

```diff
@@ -0,0 +1,39 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}: let
+  inherit (lib.options) mkEnableOption mkOption;
+  inherit (lib.types) submodule listOf str;
+  inherit (lib.nvim.types) mkGrammarOption mkPluginSetupOption;
+in {
+  options.vim.notes.neorg = {
+    enable = mkEnableOption "neorg: Neovim plugin for Neorg";
+
+    setupOpts = mkPluginSetupOption "Neorg" {
+      load = {
+        "core.defaults" = mkOption {
+          default = {};
+
+          type = submodule {
+            options = {
+              enable = mkEnableOption "A wrapper to interface with several different completion engines.";
```

Comment:
`mkEnableOption` prepends "Whether to enable" before the given string. You may remove the article.